outlines/serve/openai_server.py

```diff
@@ -0,0 +1,785 @@
+import argparse
+import asyncio
+import codecs
+import json
+import time
+from http import HTTPStatus
+from typing import AsyncGenerator, Dict, List, Optional, Tuple, Union
+
+from aioprometheus import MetricsMiddleware
```

Comment:
Please ensure any new dependencies are added are part of `pyproject.toml`

Reply:
Wouldn't they be installed when installing vLLM?

Reply:
Yes, I should have pulled before comparing to me local `openai_server.py`...